### PR TITLE
marching towards a first pass at the annotations API

### PR DESCRIPTION
**Pull request by bcoe**
- [x] set a billing email address so that we can use it in our OAuth client.
- [x] begin shipping the correct (now internal) version of the npm website.
- [x] test on beta branch.
- [x] test on fresh box

### CODE REVIEW

**Comment by nexdrew** [comment]
I have tested this against a fresh box using the npmE AMI (where I had to `npm i -g npme --ignore-scripts` in order to use `npme addon tonic`) and verified the Tonic addon works (albeit without the `tonicdev.ico` image). 👍

File: replicated/newww/start.sh [hunk 1]
```diff
@@ -7,12 +7,14 @@
...
-ln -f -s /etc/npme/data/npm-explicit-installs /etc/npme/node_modules/newww/node_modules
+rm -rf /etc/npme/node_modules/@npm/website/node_modules/npm-explicit-installs
+ln -f -s /etc/npme/data/npm-explicit-installs /etc/npme/node_modules/@npm/website/node_modules
 
 { cd /etc/npme/node_modules/@npm/npmo-web-proxy; node ./bin/npmo-web-proxy.js start --auth-host=$AUTH_HOST; } &
-{ cd /etc/npme/node_modules/newww; node server.js; } &
+{ cd /etc/npme/node_modules/@npm/website; node server.js; } &
+{ cd /etc/npme/node_modules/oauth2-server-pg; npm run migrate -- up; node bin/oauth2-server-pg.js start --private-routes; } &
 { cd /etc/npme/node_modules/@npm/npmo-auth-callbacks; node bin/npmo-auth-callbacks.js start --certificate=$CERTIFICATE --redis=$REDIS_URL --entity-id=$ENTITY_ID --assert-endpoint=$ASSERT_ENDPOINT --logout-endpoint=$LOGOUT_ENDPOINT --nameid-format=$NAMEID_FORMAT --sso-login-url=$SSO_LOGIN_URL --sso-logout-url=$SSO_LOGOUT_URL; } &
+{ cd /etc/npme/node_modules/@npm/annotation-api; node bin/annotation-api.js start --redis-url=$REDIS_URL; } &
```
Comment (nexdrew) [inline replicated/newww/start.sh]: This `newww` container is getting pretty heavy. :) But it makes sense to colocate all of these.

Comment (bcoe) [inline replicated/newww/start.sh]: yeah, just not quite ready to toss another container in the mix yet; perhaps when we start shipping user-acl?

**Comment by nexdrew** [comment]
Rebased to pick up "Enterprise" rebranding, released as Unstable 279. Still looks good.

**Comment by nexdrew** [comment]
Released to Stable channel with version `9824cc1 (288)`

**Pull request merged**